src/attr/_make.py

```diff
@@ -128,8 +128,12 @@ def attrib(
 
     :type validator: ``callable`` or a ``list`` of ``callable``\\ s.
 
-    :param bool repr: Include this attribute in the generated ``__repr__``
-        method.
+    :param repr: Include this attribute in the generated ``__repr__``
```

Comment:
"is formatted in the" ... ? "resulting repr" presumably?

Reply:
fixed in 6d37c8a4eaa5ffa39077ac5a64e70687b5ccde53

Reply:
fixed in fixed in 6d37c8a4eaa5ffa39077ac5a64e70687b5ccde53